### DISCUSSION TITLE
Back action in app drawer

### DISCUSSION THREAD

**Discussion by Slion** [LawnchairLauncher/lawnchair#3438] [Q&A]
On Huawei P30 Pro using full screen navigation gestures it seems that the back gesture is not working when in the app drawer. Instead to close the app drawer I need to scroll back all the way up to pull it down. Is there no way to ge the back button/gesture working there?

That also means that once the keyboard is shown I can close it without closing the drawer.

**Comment by SuperDragonXD** [comment]
Hello. I know that this is a bit late, but closing this discussion as this is a bit outdated. If you are still experiencing this issue, please create a new *issue* at this repo.
